src/format.rs

```diff
@@ -41,36 +41,63 @@ impl fmt::Display for FormattedDuration {
     }
 }
 
+// `HumanDuration` should be as intuitively understandable as possible.
+// So we want to round, not truncate (otherwise 1 hour and 59 minutes
```

Comment:
Dangling parenthesis; if the parenthesis was intended to be closed near the end of the sentence, perhaps use a semi-colon or some other way instead of the long parenthetical?